easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'ProbABEL'
+version = '0.5.0'
+
+homepage = "http://www.genabel.org/packages/ProbABEL"
+description = """Tool for genome-wide association 
+analysis of imputed genetic data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.genabel.org/sites/default/files/software/']
+sources = ['probabel-%(version)s.tar.gz']
+
+dependencies = [
+    ('Eigen', '3.2.6')
+]
+
+configopts = '--disable-latex-doc'
+
+sanity_check_paths = {
+    'files': ['bin/probabel', 'bin/pacoxph',
+	'bin/palinear', 'bin/palogist','etc/probabel_config.cfg.example'],
+    'dirs': ['share'],
+}
+
+moduleclass = 'bio'
+
+## You need to change the file /etc/probabel_config.cfg.example
+## to etc/probabel_config.cfg in the install directory for
+## it to work.
```

Comment:
@rtripath89 can you clarify this?

is it just copying `etc/probabel_config.cfg.example` to `etc/probabel_config.cfg`, or does the file need to be edited as well?

if so, what needs to be changed? what happens if this is not done?